plugin/path_static_account_secrets.go

```diff
@@ -0,0 +1,110 @@
+package gcpsecrets
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func pathStaticAccountSecretServiceAccountKey(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern:    fmt.Sprintf("%s/%s/key", staticAccountPathPrefix, framework.GenericNameRegex("static_account")),
+		Deprecated: true,
+		Fields: map[string]*framework.FieldSchema{
+			"static_account": {
+				Type:        framework.TypeString,
+				Description: "Required. Name of the static_account.",
+			},
+			"key_algorithm": {
+				Type:        framework.TypeString,
+				Description: fmt.Sprintf(`Private key algorithm for service account key - defaults to %s"`, keyAlgorithmRSA2k),
+				Default:     keyAlgorithmRSA2k,
+			},
+			"key_type": {
+				Type:        framework.TypeString,
+				Description: fmt.Sprintf(`Private key type for service account key - defaults to %s"`, privateKeyTypeJson),
+				Default:     privateKeyTypeJson,
+			},
+			"ttl": {
+				Type:        framework.TypeDurationSecond,
+				Description: "Lifetime of the service account key",
+			},
+		},
+		ExistenceCheck: b.pathStaticAccountExistenceCheck,
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.ReadOperation:   &framework.PathOperation{Callback: b.pathStaticAccountSecretKey},
+			logical.UpdateOperation: &framework.PathOperation{Callback: b.pathStaticAccountSecretKey},
+		},
+		HelpSynopsis:    pathServiceAccountKeySyn,
+		HelpDescription: pathServiceAccountKeyDesc,
+	}
+}
+
+func pathStaticAccountSecretAccessToken(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern:    fmt.Sprintf("%s/%s/token", staticAccountPathPrefix, framework.GenericNameRegex("static_account")),
+		Deprecated: true,
```

Comment:
Is this supposed to be marked as deprecated?

Reply:
Removed the deprecation.